source/hardware/controllers/numark_mixtrack_platinum_fx.rst

```diff
@@ -0,0 +1,866 @@
+.. Credit to PopHippy for creating the original PDF file.
+
+Numark Mixtrack Platinum FX
+
```

Comment:
This blank line before the dash line indicating a header makes the page generator pick **Overview** as page title.
Please remove.
![image](https://github.com/mixxxdj/manual/assets/5934199/088cef39-c869-4d5c-b767-e5b4223066ca)

Reply:
Updated header.